ui-frontend/packages/catalog-ui-search/src/main/webapp/component/table-export/table-export.tsx

```diff
@@ -81,6 +82,9 @@ function getColumnOrder(): string[] {
 function getHiddenFields(): string[] {
   return user.get('user').get('preferences').get('columnHide')
 }
+function getSorts() {
+  return user.get('user').get('preferences').get('resultSort')
```

Comment:
With this change, will the exported results respect the query sorts if the user has not set any result sorts?

Reply:
It does now 🙃 thanks for catching that!